src/main/java/uk/ac/ebi/embl/converter/gff3toff/Gff3ToFFConverter.java

```diff
@@ -24,18 +24,56 @@ public void convert(BufferedReader reader, BufferedWriter writer)
         try (GFF3FileReader gff3Reader = new GFF3FileReader(reader)) {
             GFF3Mapper mapper = new GFF3Mapper();
             gff3Reader.readHeader();
-            GFF3Annotation annotation;
-            while ((annotation = gff3Reader.readAnnotation()) != null) {
-                EmblEntryWriter entryWriter = new EmblEntryWriter(mapper.mapGFF3ToEntry(annotation));
-                entryWriter.setShowAcStartLine(false);
+            GFF3Annotation previousAnnotation = null;
+            GFF3Annotation currentAnnotation;
+            // The GFF3 reader returns an annotation every time it encounters a '###' directive or when the accession
+            // number of a feature changes.
+            // Since an EMBL Flat File (FF) cannot have multiple entries with the same accession, all annotations
+            // pertaining to the same accession
+            // must be merged into a single EmblEntry before being written to the output.
+            while ((currentAnnotation = gff3Reader.readAnnotation()) != null) {
+                // Merge the annotations if the accession is the same
                 try {
-                    entryWriter.write(writer);
-                } catch (IOException e) {
-                    throw new WriteException(e);
+                    if (isSameAnnotation(previousAnnotation, currentAnnotation)) {
+                        previousAnnotation.merge(currentAnnotation);
+                    } else {
+                        // The accession is different, so write the previous annotation to EMBL
+                        if (previousAnnotation != null) writeEntry(mapper, previousAnnotation, writer);
+                        previousAnnotation = currentAnnotation;
+                    }
+                } catch (NoGFF3AccessionException e) {
+                    previousAnnotation.merge(currentAnnotation);
                 }
```

Comment:
Not sure if we need this:
```
} catch (NoGFF3AccessionException e) {
         previousAnnotation.merge(currentAnnotation);
 }
```

Reply:
This happens because either the new annotation is empty or the previous annotation is empty or both are empty. merging them is the easiest way to end up with the one that may have information.